### ISSUE TITLE
All losses are Zero. How should we USE finetuning?

### ISSUE THREAD

**Issue by Msameim181** [open-mmlab/mmpose#1806]
I'm using a hand dataset that is different from other hand images, so the MMDetection would not detect that. So I train it with YOLOX and I got high accuracy. 
And then I want to use hand Keypoint to detect keypoint on hand images. I use Label-studio and fixed annotation to coco-format (or should we better say 'FreiHandDataset'). The Training Starts but no improvement at all.

I use this [res50_freihand2d_224x224.py](./configs/hand/2d_kpt_sview_rgb_img/topdown_heatmap/freihand2d/res50_freihand2d_224x224.py) config and I load this Model [res50_freihand_224x224-ff0799bc_20200914.pth]() for pre-train model.

The first question is how to use the YOLOX I trained in this training? Should I use it? 
Second, Why is this going nowhere?

![image](https://user-images.githubusercontent.com/25700939/201541690-213052d2-fc09-456a-8e4c-123761c35deb.png)

![image](https://user-images.githubusercontent.com/25700939/201542249-ed3744ce-8e14-4aea-841c-aa3df053ab25.png)

![image](https://user-images.githubusercontent.com/25700939/201542214-75dc64dc-6cfc-41fd-827b-a353e08b29f1.png)

![image](https://user-images.githubusercontent.com/25700939/201542231-cd9dc66c-7802-4d04-8981-efd13d0ac717.png)

**Comment by ly015** [comment]
Thanks for using mmpose. Could you please share the full configs used in the experiment?
And could you please check if the bounding boxes obtained by either Label Studio or YOLOX have been added to the annotation files for the keypoint detection task? It seems that you are using a top-down keypoint model so the bbox information is necessary.

**Comment by Msameim181** [comment]
All modified config was shown in images. I used this this [res50_freihand2d_224x224.py](./configs/hand/2d_kpt_sview_rgb_img/topdown_heatmap/freihand2d/res50_freihand2d_224x224.py) config for base.

All Data that is used have bbox and 21 keypoints for hand. I use the same JSON file for training the YOLOX.

**Comment by jin-s13** [comment]
The coordination of keypoints should be in pixel.

Normally we will use the ground-truth bounding box for training. And during inference, we can use YOLOX detection results.

**Comment by Msameim181** [comment]
Thanks for the update.
I Changed my dataset format to COCO and use this format as DataClass in the config. Now I'm using HRNet and everything works fine.
Thanks.

**Comment by Msameim181** [comment]
And for The Final Question, How could we Contribute to the Pre-train models and datasets?
